### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by german-e-mas**
This addresses #12 . The Readme file is updated with the steps I made to setup the environment and run the Website locally.

ptal @basicNew and @adamantivm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,73 @@
...
+With Ruby installed and set in the correct version, enter the following:
 
-`bundle install`
+```
+gem install bundler
+bundle install
+```
 
-## Contributing
+Some gems might require additional dependencies, such as *NodeJS*. Make sure to install those accordingly.
```
Comment (basicNew) [inline README.md]: Maybe I'm missing something, but why did you remove the instructions to install node and replaced it with this text?

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,5 @@
 source "https://rubygems.org"
+ruby "2.2.2"
```
Comment (basicNew) [inline Gemfile]: Was this upgrade 2.0 -> 2.2 necessary?

Comment (german-e-mas) [reply Gemfile]: My bad, it was using 2.2.0 before and I changed it to 2.2.2. I'll test and revert it back to the original 2.2.0.

File: README.md (hunk 1 above)
Comment (german-e-mas) [reply README.md]: @basicNew , I'll reinclude the node installation line.

I added this line to be more general about the dependencies. I didn't need to install anything, but probably because I already have those dependencies installed (like Node).

Is there a way to list all the dependencies required for the gems?

Comment (basicNew) [reply README.md]: Not that I know when it comes to system dependencies (i.e. those installed via `apt-get`)

**Comment by german-e-mas** [comment]
I addressed the comments @basicNew . Changed the ruby version back to `2.2.0`, included the NodeJS dependency and the `ruby-gemset` for RVM.

File: README.md [hunk 3]
```diff
@@ -1,45 +1,80 @@
...
+```
 
-To run the website in your computer
+## Run the Website
 
-`middleman`
+The following command makes the website run in the localhost. Check the `config.rb` file for the address.
+
+```
+middleman
```
Comment (basicNew) [inline README.md]: Can we just add a single way of running things? The two ways of running that you are listing are slightly different, so different setups (with or without RVM) may have different results.

File: README.md [hunk 4]
```diff
@@ -1,45 +1,80 @@
...
+```
+bundle exec middleman
+```
+
+## Build the Website
+
+To build a static version of the Website, run:
+
+```
+middleman build
```
Comment (basicNew) [inline README.md]: Sames as above

File: README.md (hunk 3 above)
Comment (german-e-mas) [reply README.md]: I'll leave the ones I used (`middleman` instead of `bundle exec middleman`)

**Comment by german-e-mas** [comment]
Thanks for the comments @basicNew ! I removed the extra commands. I'll proceed to merge this.
